### ISSUE TITLE
Problem with npolynb_samedomain

### ISSUE THREAD

**Issue by xajezade** [rquey/neper#72]
Hi there,

I am having a problem using this option. I have a multi-scale tesselation file (in 3D) and I need to perform neighbors statistics but it seems to fail with a segmentation fault.

`neper -T -loadtess "final.tess" -statpoly npolys_samedomain`

however, this command works:

`neper -T -loadtess "final.tess" -statcell nseeds`

Kindest Regards,
Ali

**Comment by xajezade** [comment]
Hi there, 

I have found the problem. when I use -loadtess, it converts a multiscale tesselation file into a single scale tesselation file. My mistake here was that I had performed `neper -T -loadtess "final.tess"` before using the `neper -T -loadtess "final.tess" -statpoly npolys_samedomain` which replaces the final.tess with another file with the same name but this time it is not a multi-scale anymore. I was expecting that `-npolys_samedomain` also works for single scale tessellation but apparantly it just works if I do this `neper -T -loadtess "final.tess" -statpoly npolys_samedomain -o anotherTess`.
